Mathlib/CategoryTheory/Enriched/FunctorCategory.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2024 Jack McKoen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jack McKoen
+-/
+import Mathlib.CategoryTheory.Functor.FunctorHom
+import Mathlib.CategoryTheory.Enriched.Basic
+
+/-!
+# Functor categories are enriched over functors to type
+
+Shows that the functor category `C ⥤ D` is enriched over `C ⥤ Type max v' v u`, for `C` a
+category in `Type u` with morphisms in `Type v`, and `D` a category with morphisms in `Type v'`.
+-/
+
+
+universe w v' v u u'
+
+open CategoryTheory MonoidalCategory Functor
+
+variable {C : Type u} [Category.{v} C] {D : Type u'} [Category.{v'} D]
+
+variable (F G : C ⥤ D)
+
+namespace CategoryTheory.Enriched.Functor
+
+@[simp]
+lemma natTransEquiv_app_app_apply (F G : C ⥤ D) (f : F ⟶ G)
```

Comment:
This lemma (and some of the other lemmas below) should probably go to the other file in this PR, as they're part of the API for the definitions introduced there.